packages/react/src/components/Forms/Select/mono-searchable.tsx

```diff
@@ -57,30 +73,39 @@ export const SelectMonoSearchable = forwardRef<SelectHandle, SubProps>(
         return;
       }
 
-      const selectedItem = downshiftReturn.selectedItem
-        ? optionToValue(downshiftReturn.selectedItem)
-        : undefined;
+      if (Array.isArray(props.options)) {
```

Comment:
it seems that this useEffect will not work when the select is controlled + async options?